eng/pipelines/templates/steps/ref-updater.yml

```diff
@@ -6,10 +6,10 @@ parameters:
   Repos: []
 
 steps:
-
+- template: /eng/common/pipelines/templates/steps/set-default-branch.yml
 - ${{ each repo in parameters.Repos }}:
   - pwsh: |
-      git clone --branch master https://github.com/azure/${{ repo }}
+      git clone --branch $(DefaultBranch) https://github.com/azure/${{ repo }}
```

Comment:
This is assuming that all the repo's have the same default branch I think we need to move the template that sets the default branch into the inner loop.

Reply:
git clone to master is not actual necessary. Changed to 
```
git clone https://github.com/azure/${{ repo }}
```
Then, we have no trouble to call `set-default-branch.yml` after this step.